wdl-lint/src/rules/shellcheck.rs

```diff
@@ -367,12 +351,18 @@ impl Visitor for ShellCheckRule {
                     let start = line_map
                         .get(&diagnostic.line)
                         .expect("shellcheck line corresponds to command line");
-                    let inner_span = {
-                        Span::new(
-                            start + diagnostic.column,
-                            diagnostic.end_column - diagnostic.column,
-                        )
+                    let len = if diagnostic.end_line > diagnostic.line {
```

Comment:
need a test for multiline diagnostic. Have to figure out how to trigger one in shellcheck first.